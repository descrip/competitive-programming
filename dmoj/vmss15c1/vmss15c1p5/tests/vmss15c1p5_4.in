9 719 977
APPL 532 16 19
internet 373 4 29
mountain 724 16 26
quack 236 9 17
reddelicious 536 5 22
reddit 901 16 20
stang 531 11 3
stromae 614 9 30
sucky 449 18 15
