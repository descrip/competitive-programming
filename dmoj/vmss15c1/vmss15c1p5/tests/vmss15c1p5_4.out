45690
APPL 0
internet 0
mountain 0
quack 0
reddelicious 37
reddit 1
stang 47
stromae 0
sucky 0
